app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -284,14 +305,14 @@ class PipelineSampleReport extends React.Component {
                         </span>
                         Taxonomy
                       </th>
-                      { this.render_column_header('NT+NR', 'ZZRPM',  'nt_aggregatescore') }
-                      { this.render_column_header('NT', 'Z',   'nt_zscore') }
-                      { this.render_column_header('NT', 'rPM', 'nt_rpm')    }
-                      { this.render_column_header('NT', 'r',   'nt_r')      }
-                      { this.render_column_header('NT', '%id', 'nt_percentidentity')    }
-                      { this.render_column_header('NT', 'AL',   'nt_alignmentlength')    }
-                      { this.render_column_header('NT', 'Log(1/E)',  'nt_neglogevalue')    }
-                      { this.render_column_header('NR', 'Z',   'nr_zscore') }
+                      { this.render_column_header('NT+NR', 'ZZRPM',  'nt_aggregatescore', 'Aggregate score') }
+                      { this.render_column_header('NT', 'Z',   'nt_zscore', 'z-score relative to background model') }
+                      { this.render_column_header('NT', 'rPM', 'nt_rpm', 'number of reads per million total input reads')    }
+                      { this.render_column_header('NT', 'r',   'nt_r', 'number of reads aligning to the species in the NCBI NT database')      }
+                      { this.render_column_header('NT', '%id', 'nt_percentidentity', 'average percent-identity of alignments')    }
+                      { this.render_column_header('NT', 'L',   'nt_alignmentlength', 'average length of alignments')    }
+                      { this.render_column_header('NT', 'Log(1/E)',  'nt_neglogevalue', 'average log-10-transformed expect value')    }
+                      { this.render_column_header('NR', 'Z',   'nr_zscore', 'number of reads aligning to the species in the NCBI NR database') }
```

Comment:
Capitalize and replace species by taxon: 'Number of reads aligning to the taxon in the NCBI NR database'